python/pandemic_simulator/environment/pandemic_env.py

```diff
@@ -169,3 +169,37 @@ def reset(self) -> PandemicObservation:
 
     def render(self, mode: str = 'human') -> bool:
         pass
+
+class PandemicGymEnv3Act(gym.ActionWrapper):
```

Comment:
Could you change it to a more clear name?

Reply:
How about PandemicGymEnvRA RA for reduced actions or ReducedPandemicGymEnv?

Reply:
`ReducedActionPandemicGymEnv`?

Reply:
I have renamed it to ReducedActionPandemicGymEnv